x-pack/plugins/ml/public/application/datavisualizer/file_based/components/import_view/import_view.js

```diff
@@ -601,34 +601,16 @@ export class ImportView extends Component {
   }
 }
 
-async function createKibanaIndexPattern(
-  indexPatternName,
-  indexPatterns,
-  timeFieldName,
-  kibanaConfig
-) {
+async function createKibanaIndexPattern(indexPatternName, indexPatterns, timeFieldName) {
```

Comment:
If I try and view the index pattern that is created here in the ML index based data visualizer, it doesn't seem to recognize if it is a time based index. Looking at the index pattern on the Stack Management index patterns page, it isn't marking the time field with the usual clock icon either:

![image](https://user-images.githubusercontent.com/7405507/93773908-880a1c00-fc18-11ea-8d65-9c617021ff1c.png)

To reproduce - upload a CSV file with a time column, import the data, and then view the index in the ML Data Visualizer.

Reply:
Confirmed fixed for the file data visualizer import in https://github.com/elastic/kibana/pull/77791/commits/dd2caff346695651f1d3f3a3b0aab1977abeb18f.